analysis/statistics/total_handling_cost.py

```diff
@@ -0,0 +1,11 @@
+from typing import Tuple
+
+from heuristic.classes import Problem, Solution
+
+
+def total_handling_cost(solution: Solution) -> Tuple[str, float]:
+    """
+    Returns instance's total handling costs.
+    """
+    return "total_handling_cost", sum(solution.routes[idx].handling_cost()
+                                      for idx in range(len(solution.routes)))
```

Comment:
Newline, and also: don't use indices more than you have to. This is better written as `sum(route.handling_cost() for route in solution.routes)`